src/components/application_manager/src/hmi_state.cc

```diff
@@ -243,30 +243,30 @@ PhoneCallHmiState::PhoneCallHmiState(std::shared_ptr<Application> app,
 
 mobile_apis::HMILevel::eType PhoneCallHmiState::hmi_level() const {
   using namespace mobile_apis;
-  if (HMILevel::INVALID_ENUM == parent_hmi_level()) {
+  if (HMILevel::INVALID_ENUM == parent_hmi_level() ||
+      HMILevel::HMI_NONE == parent_hmi_level()) {
     return parent_hmi_level();
   }
-  return std::max(parent_hmi_level(), max_hmi_level());
-}
-
-mobile_apis::HMILevel::eType PhoneCallHmiState::max_hmi_level() const {
-  using namespace helpers;
-  using namespace mobile_apis;
 
   if (WindowType::WIDGET == window_type()) {
     return std::max(HMILevel::HMI_FULL, parent_max_hmi_level());
   }
 
-  auto expected = HMILevel::HMI_FULL;
+  auto expected = HMILevel::HMI_BACKGROUND;
```

Comment:
Is there a reason that non-media apps should be deactivated by this event? I think they should still be allowed to be in FULL, as the phone call could be handled in the background. In addition, this was set up to follow previous behavior, so I don't know if it should be changed without a decent reason.

Reply:
That said, I could see an argument to not change the HMI level at all, since the phone call could be started in the background without ever deactivating the app. Perhaps this event should ONLY affect audioStreamingState?

Reply:
Currently all other types of apps are deactivated by this event, and so I feel that non-media apps should be deactivated as well. Do you agree that having all apps deactivated is better than having most deactivated?

I prefer the idea of having Phone Call event only affect audible status from an HMI perspective, I think it makes the event simpler to use, but this would be a larger change. I couldn't think of a great reason to make this change, could you?

Reply:
Turns out this is already part of an accepted proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0264-Separating-the-change-of-Audible-status-and-the-change-of-HMI-Status.md